src/style.css

```
@@ -1 +1,69 @@
 /* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  --color-blue: #00acdc;
+  cursor: pointer;
+}
+
+.header {
+  display: flex;
+  margin: 0 50px;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 12px;
+  color: #000;
+  justify-content: space-between;
+
+}
+
+.logo__image {
+
+  width: 40px;
+  height: 40px;
+  padding: 10px 0;
+
+}
+
+.nav__list {
+  height: 100%;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+  align-items: center;
+
+}
+
+.nav__link {
+  font-family: "Roboto", sans-serif;
+  color: #000;
+  padding: 23px 0;
+  margin-left: 20px;
+  text-decoration: none;
+
+}
+
+a:hover {
+  color: var(--color-blue);
+}
+
+a:active,
+.is-active {
+  color: var(--color-blue);
+  margin-left: 0;
+  position: relative;
+  display: flex;
+  align-items: end;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  height: 4px;
+  width: 37px;
```

```suggestion
  width: 100%;
```
different links will have different width